src/backend/wallet_connect.nim

```diff
@@ -22,15 +22,15 @@ proc isSuccessResponse(rpcResponse: RpcResponse[JsonNode]): bool =
 proc addSession*(sessionJson: string): bool =
   try:
     let rpcRes = addWalletConnectSession(sessionJson)
-    return isSuccessResponse(rpcRes):
+    return isSuccessResponse(rpcRes)
```

Comment:
:pepe_think:

Reply:
yeah! i have no idea how this worked before!